source/client.c

```diff
@@ -57,6 +61,8 @@ int aws_mqtt_client_init(
 
 void aws_mqtt_client_clean_up(struct aws_mqtt_client *client) {
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "client=%p Cleaning up MQTT client", (void *)client);
```

Comment:
The first two log messages have "client=%p" and later ones use "id=%p"
Should probably stick with "id=%p". The client already has its own subject string, and "id" is we've been doing in other systems

Reply:
These are two different things, id= is used when referring to a connection, not the client.